Developers/Writing_Bears.rst

```diff
@@ -223,22 +223,22 @@ So let's see how you could tell coala which Bears to run before yours:
 
     class DependentBear(LocalBear):
 
+        BEAR_DEPS = {OtherBear}
+
         def run(self, filename, file, dependency_results):
             results = dependency_results[OtherBear.name]
 
-        @staticmethod
-        def BEAR_DEPS():
-            return [OtherBear]
 
 As you can see we have a :meth:`~coalib.bears.Bear.Bear.BEAR_DEPS`
```

Comment:
@sils Actually there is a problem here. `:meth:` should be replaced with `:attr:`. Should I create a new issue?